bifrost/release.go

```diff
@@ -236,33 +241,47 @@ func (r *Release) TimedOut() error {
 ///////
 
 // UnlockRootLock deletes the Lock File for the release
-func (r *Release) UnlockRoot(s3c aws.S3API) error {
-	return s3.ReleaseLock(s3c, r.Bucket, r.RootLockPath(), *r.UUID)
+func (r *Release) UnlockRoot(s3c aws.S3API, locker Locker, lockTableName string) error {
+	if err := s3.ReleaseLock(s3c, r.Bucket, r.RootLockPath(), *r.UUID); err != nil {
+		return err
+	}
+
+	return locker.ReleaseLock(lockTableName, *r.RootLockPath(), *r.UUID)
 }
 
 // GrabLock retrieves the Lock returns LockExistsError, or LockError
-func (r *Release) GrabLocks(s3c aws.S3API) error {
+func (r *Release) GrabLocks(s3c aws.S3API, locker Locker, lockTableName string) error {
 	if err := r.CheckUserLock(s3c, *r.UserLockPath()); err != nil {
 		return err
 	}
 
-	if err := r.GrabReleaseLock(s3c); err != nil {
+	if err := r.GrabReleaseLock(s3c, locker, lockTableName); err != nil {
```

Comment:
I dont think we will need to grab the release lock in the dynamoDB table. This lock is specifically here to stop someone replaying the same release over and over. If the Root lock works this will be impossible.